nest-forms-backend/src/nases/dtos/requests.dto.ts

```diff
@@ -305,6 +305,10 @@ export class GetFormResponseDto {
   @IsNotEmpty()
   @IsString()
   declare jsonVersion: string
+
+  @ApiProperty()
+  @IsBoolean()
+  declare requiresMigration: boolean
```

Comment:
shouldn't dtos also use the ! syntax? or will we stick with it only in the V2 files and keep it consistent in the old ones with declare?